packages/playwright-core/src/server/chromium/crPage.ts

```diff
@@ -432,6 +432,8 @@ class FrameSession {
       this._firstNonInitialNavigationCommittedFulfill = f;
       this._firstNonInitialNavigationCommittedReject = r;
     });
+    // We do not always await this promise.
```

Comment:
Without that this Promise ends up in Promise unhandled rejection errors - since we set an error to it (inside FrameSession.dispose) but never read the error.

Reply:
We wait for it in `FrameSession._initialize` which is called for both main frame and OOPIFs soon after the session is created. I guess there may be an exception before this promise is added to the list of awaited promises, but does it happen in reality? If it does, we probably need to update wkPage too.